kiwi/oci_tools/buildah.py

```diff
@@ -161,7 +161,12 @@ def sync_rootfs(self, root_dir, exclude_list=None):
         self._sync_data(
             ''.join([root_dir, os.sep]), self.oci_root_dir,
             exclude_list=exclude_list,
-            options=Defaults.get_sync_options() + ['--delete']
+            options=Defaults.get_sync_options() + [
+                '--filter="-x! user.*"',
+                '--filter="-x! security.ima*"',
+                '--filter="-x! security.capability*"',
```

Comment:
If we don't sync caps, then things like unprivileged ping wouldn't work, right?

Reply:
I think I can't follow you. The "security.capability*" matches what you described as "caps" or is this a different namespace ?

actually the filter definition of "-x! namespace" says filter out everything but namespace (negated match). I hope that multiple of these filters can be combined. And yes this needs testing, I haven't done this before with rsync

So which namespace are you referring to exactly ?

Thanks

Reply:
I think I didn't understand what this option does. Are you saying we're syncing this attribute in?

Reply:
yes this is at least how I understood it when reading man rsync

```
 To skip all namespaces except  the  user  namespace,  you  could
              specify a negated-user match:

              --filter=’-x! user.*’

```

Reply:
Sounds good to me, then.